bundles/tools.vitruv.domains.java/src/tools/vitruv/domains/java/JavaDomain.xtend

```diff
@@ -1,64 +1,18 @@
 package tools.vitruv.domains.java
 
 import static tools.vitruv.domains.java.JavaNamespace.*
-import org.eclipse.emf.ecore.resource.Resource
-import org.emftext.language.java.resource.JavaSourceOrClassFileResourceFactoryImpl
-import java.util.Map
-import org.emftext.language.java.resource.java.IJavaOptions
 import java.util.List
 import tools.vitruv.framework.domains.AbstractVitruvDomain
 
 class JavaDomain extends AbstractVitruvDomain {
 	static final String METAMODEL_NAME = "Java"
-	boolean shouldTransitivelyPropagateChanges = false
 	
 	protected new() {
 		this(METAMODEL_NAME)
 	}
 	
 	protected new(String name) {
 		super(name, ROOT_PACKAGE, List.of(FILE_EXTENSION))
-		// Register factory for class and Java files in case of not running as plugin
-		Resource.Factory.Registry.INSTANCE.getExtensionToFactoryMap().put("java", new JavaSourceOrClassFileResourceFactoryImpl())
-		Resource.Factory.Registry.INSTANCE.getExtensionToFactoryMap().put("class", new JavaSourceOrClassFileResourceFactoryImpl())
-		// This is necessary to resolve classes from standard library (e.g. Object, List etc.) 
-		JamoppLibraryHelper.registerStdLib
```

Comment:
Why don't we need to register the standard libraries anymore? As we do the registration already in every test case where Java is used?

Reply:
Yes, there was a bunch of complicated logic resetting the Java classpath, registering the standard library agin and sometimes also setting up resources for non-OSGI-environments. These PRs unify it and make the code in the domain (which will be removed anyway) obsolete. In a subsequent PR, there will be a proper utility class for setting up Java, which is then used by all tests requiring Java to be set up properly.